movement/movement.c

```diff
@@ -155,13 +155,7 @@ void movement_request_tick_frequency(uint8_t freq) {
     if (freq == 128) return; // Movement uses the 128 Hz tick internally
 
     // disable all callbacks except the 128 Hz one
-#if __EMSCRIPTEN__
-    for (int i = 1; i < 128; i = i << 1) {
-        watch_rtc_disable_periodic_callback(i);
-    }
-#else
-    RTC->MODE2.INTENCLR.reg = 0xFE;
-#endif
+    watch_rtc_disable_matching_periodic_callbacks(0b01111111);
```

Comment:
aha! I think this needs to be 0b11111110. Was getting some funky behavior in the beats clock, it's backwards from what you expect but the low bit is the high frequency tick, and the high bit is the 1 Hz tick.

Reply:
Ah that makes sense. I wonder if I should modify the implementation to accept this "intent", though? It makes more sense to me this way at least.

Reply:
I dunno, the joy of this one is that it's one line on the hardware, `RTC->MODE2.INTENCLR.reg = mask`; if we mirror it to make it feel right, that'll end up requiring additional bit twiddling on the hardware side. It's also used rarely enough that as long as we document it, I think it's fine if it's goofy.